.github/workflows/build_and_test.yml

```diff
@@ -43,14 +43,14 @@ jobs:
     with:
       docker-image: ${{ needs.build.outputs.docker-image }}
       runner: linux.8xlarge.nvidia.gpu
-      timeout-minutes: 300
+      timeout-minutes: 180
       disable-xrt: 1
     secrets:
       gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
 
   test-cpu-coverage:
     name: "Collect CPU test coverage"
-    if: github.event_name == 'push' && github.event.ref == 'refs/heads/master'
+    # if: github.event_name == 'push' && github.event.ref == 'refs/heads/master'
```

Comment:
Btw, make sure you rebase the master, i pushed fix for coverage test yesterday.

Reply:
Thanks for the reminder, yes I think this PR includes your fix to the coverage test